Dockerfile.tmpl

```diff
@@ -107,24 +108,15 @@ RUN conda config --add channels nvidia && \
     /tmp/clean-layer.sh
 
 # Install spacy
+# b/232247930: uninstall pyarrow to avoid double installation with the GPU specific version.
 {{ if eq .Accelerator "gpu" }}
-RUN mamba install -y -c conda-forge spacy cupy cuda-version=$CUDA_MAJOR_VERSION.$CUDA_MINOR_VERSION && \
+RUN pip uninstall -y pyarrow && \
+    conda install -y -c conda-forge spacy cudf=24.4 cuml=24.4 cupy cuda-version=$CUDA_MAJOR_VERSION.$CUDA_MINOR_VERSION && \
```

Comment:
Let's use >=24.4 so that it can upgrade over time.

Reply:
d.